frontend/src/App.js

```diff
@@ -8,9 +8,10 @@ import Box from "@mui/material/Box";
 
 import Navbar from "./components/Navbar";
 import PrivateRoute from "./components/PrivateRoute.js";
+import Home from "./pages/Home";

```

Comment:
Hi, if possible can help name it to "HomePage" and "RoomPage"? Thanks!

Reply:
Sure, good catch!